backend/src/models.py

```diff
@@ -9,4 +9,4 @@ class Rule(models.Model):
     amount = models.CharField(max_length=50, null=True, blank=True)
    
     def __str__(self):
-        return self.rule_name
\ No newline at end of file
+        return self.rule_name
```

Comment:
Since we've renamed "rule_name" to "name" shouldn't this line be "return self.name"?

Reply:
Fair. Probably snuck in due to a git conflict.